modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -132,11 +163,33 @@ panda  User
 ? Confirm deletion of role "red"?  This action will remove all associated ACLs and unassign role members Yes
 Successfully deleted role "red"
 ----
+--
+Redpanda Console::
++
+--
+Administrators use Redpanda Console to delete an existing role by following these steps:
+
+1. Select `Security` from the left navigation menu.
+
+2. Click the role you wish to delete. This will show all currently assigned permissions (ACLs) and principals (users).
+
+3. Click *Delete*.
+
+4. Redpanda Console shows you a prompt asking you to confirm deletion of the role. The prompt differs based on whether you have principals assigned to the role or not. If you have principals assigned to the role, you must enter the text `delete` in the input field where prompted before you may continue.
```

Comment:
I believe this was changed to require to type in the name of the role:

![image](https://github.com/redpanda-data/docs/assets/2081100/b3bb1175-9086-40c8-8bff-319dde5a7f46)

Reply:
Fixed the wording!